Code Quality: Make meta more modern™ (#8)
RiversideValley/.github: .github/CODE_OF_CONDUCT.md

Author: Should remove for Riverside probably, could keep as a joke though.

- [ ] 🫣 Deliberate intimidation, stalking or following (online or in person).
- [ ] 🙊 Advocating for, or encouraging, any of the above behavior.
- [ ] ⁉️ Sustained disruption of community events, including talks and presentations.
- [ ] 🪟 Negative comparison of Rebound to Rectify11 can be counted as harrasment.

> - [ ] 🖕 ACRYLIC IS ABSOLUTELY NOT PERMITTED AT ALL AND YOU WILL BE BANNED! 

Author: > 🖕 ACRYLIC IS ABSOLUTELY NOT PERMITTED AT ALL AND YOU WILL BE BANNED! 

Should change the emoji.

```suggestion
- [ ] 💢 ACRYLIC IS ABSOLUTELY NOT PERMITTED IN ANY OF OUR APPS!
```
